src/main/java/org/jabref/logic/util/io/FileUtil.java

```diff
@@ -331,8 +331,8 @@ public static String createDirNameFromPattern(BibDatabase database, BibEntry ent
      * @return the path to the first file that matches the defined conditions
      */
     public static Optional<Path> find(String filename, Path rootDirectory) {
-        try {
-            return Files.walk(rootDirectory)
+        try (Stream<Path> pathStream = Files.walk(rootDirectory)) {
```

Comment:
Can you please check if `walk` may also throw an `IOUncheckedEx` and if yes, add it to the catch block.

Reply:
Checked the JavaDoc and the sources... does not seem so.